templates/google_tag_manager.html

```diff
@@ -0,0 +1,17 @@
+<%!
+  from django.conf import settings
```

Comment:
Now that we're defining `google_tag_manager_id` locally,
we can drop this import.